### ISSUE TITLE
[FEATURE REQUEST] Volcano emission switch does not allow for climatological flags and is not leap year friendly

### ISSUE THREAD

**Issue by melaniehammer** [geoschem/geos-chem#500]
In the newer versions of GC (12 and up) the volcanic emissions are defined in the "switch" part of the HEMCO_Config.rc file:
```
### NOTE: Until further notice, deactivate the AeroCom volcano extension (bmy, 5/3/18)
117     Volcano                : on    SO2
    --> Volcano_Source         :       AeroCom
    --> Volcano_Table          :       $ROOT/VOLCANO/v2019-04/$YYYY/$MM/so2_volcanic_emissions_Carns.$YYYY$MM$DD.rc
```
which means there are no options to use climatological flags for dates where emissions are not yet available. For example, I am running a simulation for Feb 2020, and there are no volcanic emissions past Jan 2020. So I set $YYYY to 2019, but when it got to Feb 29th, my simulation crashed because there is no Feb 29th 2019. The volcanic emissions should be moved to the actual emissions portion of the HEMCO_Config.rc so climatological flags can be set and you can just use the closest available year data if emissions for the year of your simulation are not available. 
ERROR output from HEMCO.log file:
```
AeroCom: reading /ExtData/HEMCO/VOLCANO/v2019-04/2019/02/so2_volcanic_emissions_Carns.20190229.rc
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
HEMCO ERROR: Cannot find /ExtData/HEMCO/VOLCANO/v2019-04/2019/02/so2_volcanic_emissions_Carns.20190229.rc
ERROR LOCATION: ReadVolcTable (hcox_volcano_mod.F90)
```

**Comment by msulprizio** [comment]
Thank you for bringing this up. We are working on this for the 13.0.0 -- see https://github.com/geoschem/geos-chem/issues/331. Right now we are preparing the initial 1-month and 1-year benchmarks to send out to developers and the GCSC. Once approved, we will send a release candidate to the community to try out and test for a month while we run a 10-year benchmark simulation (for use in a new stratospheric benchmark). Once the 10-year benchmark is approved and any bugs/issues that are reported during the release candidate period is resolved, we will officially release 13.0.0. We are hoping to have this update ready for the official 13.0.0 release (in about 1-2 months), as we still have to generate the climatology files.

**Comment by melaniehammer** [comment]
Great, thank you for your quick response!

-Melanie.

**Comment by yantosca** [comment]
I will close out this issue.  As @msulprizio mentioned, this will be resolved for the 13.0.0 release (cf feature request #331), once we get the climatology fields from the 10-yr benchmarks.

**Comment by msulprizio** [comment]
I am reopening this issue. The [feature request linked above](https://github.com/geoschem/geos-chem/issues/331) addressed only chemistry input files (e.g. oxidant fields, prod/loss rates). We will aim to provide a climatology option for the volcano emissions by [13.2.0](https://github.com/geoschem/geos-chem/milestone/18).

**Comment by msulprizio** [comment]
This is still an active issue. I am changing to a Feature Request to ensure we add this item in an upcoming version.

**Comment by yantosca** [comment]
Feature request #833 has now been closed, so we can close this issue as well.  The volcano climatology has been merged into the 13.3.0 development stream and will be released with version 13.3.0.
